app/MMB/src/components/menu/MenuItem.js

```diff
@@ -10,22 +10,27 @@ import {
   Touchable,
   AdaptableLayout,
   Icon,
+  type StylePropType,
 } from '@kiwicom/mobile-shared';
 import { Translation } from '@kiwicom/mobile-localization';
 
 import MenuItemIcon from './MenuItemIcon';
 
 type Props = {|
   title: React.Element<typeof Translation>,
-  onPress: () => void,
   isActive: boolean,
 
+  onPress?: () => void,
   // icon on the left
   icon?: React.Element<typeof TextIcon>,
 
   // icon on the right (arrow by default)
   actionIcon?: React.Element<typeof TextIcon>,
   description?: React.Element<typeof Translation>,
+
+  // style props
+  titleStyle?: StylePropType,
+  descriptionStyle?: StylePropType,
```

Comment:
Why is it needed? This component is opinionated about styling to prevent style hotfixing...

Reply:
So I should create a new one then? Since my menu item has completely different styling than the existing one

Reply:
> my menu item has completely different styling than the existing one

Then it's completely different component? Can you please share screens for comparison? This PR is about _"MMB: Make MenuItem onPress optional"_ but you are tweaking styles here... 🤔